src/js_realm.hpp

```diff
@@ -212,6 +213,7 @@ struct RealmClass : ClassDefinition<T, SharedRealm> {
         {"create", wrap<Realm::create>},
         {"delete", wrap<Realm::delete_one>},
         {"deleteAll", wrap<Realm::delete_all>},
+        {"isValid", wrap<Realm::is_object_valid>},
```

Comment:
This should be a static method: `Realm.isValid()`. That way you don't need a reference to the realm instance to make these checks, and this method doesn't verify that the object passed in is a part of this realm anyways.